src/Command/MeiliSearchImportCommand.php

```diff
@@ -154,10 +154,9 @@ function ($entity) {
                 }
 
                 ++$page;
-                $repository->clear();
```

Comment:
Why do you remove this lin?

Reply:
The [`clear`](https://github.com/doctrine/orm/blob/2.8.x/lib/Doctrine/ORM/EntityRepository.php#L146) method of the `EntityRepository` is set `deprecated` without any replacement method. So there is no other chance than to remove this line.

As the PR [calls the `clear` method](https://github.com/meilisearch/meilisearch-symfony/pull/75/files#diff-847a91fa82351c9ee188c77627b83af2d7a0aa243b35034200eb429e20c4bcfaR159) of the entity manager after the loop I thought a separate call for each repository is not needed.